Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -105,6 +105,7 @@
     - DoorBumpOpener
   - type: Input
     context: "human"
+  - type: PayloadCase
```

Comment:
This still needs to be removed

Reply:
haha i removed that. But then when I merged the branches it happened, now I'll fix it.\